### PR TITLE
Update CrUX guides with Core Web Vitals

### PR DESCRIPTION
**Pull request by rviscomi**
Progress on #2365

Changes proposed in this pull request:

- updated screenshots to latest PSI/Dashboard/BQ UI
- updated thresholds and percentiles
- updated metrics

### CODE REVIEW

**Comment by kaycebasques** [comment]
@rviscomi please re-assign to me when ready for review

**Comment by rviscomi** [comment]
Ready for review. I left out the BigQuery guide pending further discussion. Wanted to unblock the other two guides.

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: post
 title: Using the CrUX Dashboard on Data Studio
 authors:
   - rviscomi
+hero: hero.png
 description: |
   Data Studio is a powerful data visualization tool that enables you to build
   dashboards on top of big data sources, like the Chrome UX Report In this
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:9]: Drive-by: Missing a period between `Report` and `In`

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 2]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+
+{% Compare 'worse', 'URL (Not supported)' %}
+```text
+https://developers.google.com/web/tools/chrome-user-experience-report/
+```
+{% endCompare %}
+
+If you omit the protocol, HTTPS is assumed. Subdomains matter, for example
+`https://developers.google.com` and `https://www.google.com` are considered to 
+be very different origins.
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:57]:
```suggestion
be different origins.
```
I don't think we want the emphasis of `very` here because it implies that there are shades/degrees of differentness in origins

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 3]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+<img class="w-screenshot" src="./dash_error.png" alt="CrUX Dashboard error message">
 
 If the origin exists, you'll be taken to the schema page for the dashboard.
 This shows you all of the fields that are included: each effective connection
 type, each form factor, the month of the dataset release, the distribution of
-performance for each supported metric (e.g., first contentful paint), and of
-course the name of the origin. There's nothing you need to do or change on this
-page, just click "Create Report" to continue.
+performance for each metric, and of course the name of the origin. There's
+nothing you need to do or change on this page, just click "Create Report" to
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:76]:
```suggestion
nothing you need to do or change on this page, just click **Create Report** to 
```
https://developers.google.com/style/ui-elements#formatting

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 4]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
 
-{% Aside %}
-Due to a quirk with Data Studio, you may sometimes see "No Data" here. This is
-normal and due to the previous month's release not being available yet. Datasets
-are released on the second Tuesday of each month.
-{% endAside %}
+Use the Core Web Vitals page to understand how the origin is experienced by 
+desktop and phone users. By default, the most recent month at the time you 
+created the dashboard is selected. To change between older or newer monthly 
+releases, use the "Month" filter at the top of the page.
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:107]:
```suggestion
releases, use the **Month** filter at the top of the page.
```

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 5]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
 
-### Device distribution
+Note that tablet is omitted from these charts by default, but if needed you 
+could remove the "No Tablet" filter in the bar chart configuration, shown
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:110]:
```suggestion
could remove the **No Tablet** filter in the bar chart configuration, shown 
```

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 6]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+in the CrUX dataset.
 
-The third and final page shows you the breakdown of effective connection types
-per month. There are five possible connection types (4G, 3G, 2G, Slow 2G, and
-Offline), but you will typically only see two or three in the chart. In the
-example above, 2G, Slow 2G, and offline are all 0%.
+<img class="w-screenshot" src="./dash_lcp.png" alt="CrUX Dashboard LCP page">
 
-![CrUX Data Studio effective connection type dashboard](./dash_ect.png)
+Atop each page is the "Device" filter, which you can use to restrict the form
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:123]:
```suggestion
Atop each page is the **Device** filter, which you can use to restrict the form 
```

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 7]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+factors included in the experience data. For example, you can drill down 
+specifically into phone experiences. This setting persists across pages.
 
-## Sharing the dashboard
+The primary visualizations on these pages are the monthly distributions of
+experiences categorized as "Good", "Needs Improvement", and "Poor". The 
+color-coded legend below the chart indicates the range of experiences included
+in the category. For example, in the screenshot above, you can see the 
+percent of good [Largest Contentful Paint](/lcp/) (LCP) experiences, which are 
+categorized as those less than 2500ms, fluctuating and getting slightly worse
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:132]: I recommend omitting a reference to a specific value (`2500ms`) unless you have some sort of system that tracks every page where specific CWV values are hardcoded (so that you can quickly update all those pages if the values ever change in the future)

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 8]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
-say, you don't have to own the site you are getting a dashboard for, so you can 
-easily use this tool to keep an eye on your competitors' sites.
+The most recent month's percentages of good and poor experiences are shown 
+above the chart along with an indicator of the percent difference from the 
+previous month. For this origin, good LCP experiences fell by 3.2% to 56.04% 
+month-over-month.
 
-![CrUX Data Studio embedding and sharing](./dash_embed.png)
+{% Aside 'caution' %}
+Due to a quirk with Data Studio, you may sometimes see "No Data" here. This is
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:141]:
```suggestion
Due to a quirk with Data Studio, you may sometimes see `No Data` here. This is
```
I'm not sure if this one is captured in the Style Guide but my convention is to use code formatting in these cases because it's literal text that you see on the screen

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 9]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
 
-Dashboards may also be embedded into websites or blog posts and retain all of
-their interactivity features.
+Additionally, for metrics like LCP and other Core Web Vitals that provide 
+explicit percentile recommendations, you'll find the "P75" metric between the 
+good and poor percentages. This value corresponds to the origin's 75th 
+percentile of user experiences. In other words, 75% of experiences are better
+than this value. One thing to note is that this applies to the overall 
+distribution across _all devices_ on the origin. Toggling specific devices 
+with the "Device" filter will not recalculate the percentile.
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:152]:
```suggestion
with the **Device** filter will not recalculate the percentile.
```

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 10]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+3800ms indicates that the true 75th percentile is somewhere between 3800ms and
+3900ms.
+
+Additionally, the BigQuery dataset uses a technique called "bin spreading" in 
+which densities of user experiences are intrinsically grouped into very coarse 
+bins of decreasing granularity. This allows us to include minute densities in 
+the tail of the distribution without having to exceed four digits of 
+precision. For example, LCP values less than 3 seconds are grouped into bins 
+200ms wide. Between 3 and 10 seconds, bins are 500ms wide, then 5000ms wide, 
+etc. Rather than having bins of varying widths, bin spreading ensures that all
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:171]:
```suggestion
200ms wide. Between 3 and 10 seconds, bins are 500ms wide. Beyond 10 seconds, bins are 5000ms wide, 
etc. Rather than having bins of varying widths, bin spreading ensures that all 
```
I stumbled over this paragraph. If my understanding is correct then I think this phrasing will make it easier to grok on first try

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 11]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+
+Additionally, the BigQuery dataset uses a technique called "bin spreading" in 
+which densities of user experiences are intrinsically grouped into very coarse 
+bins of decreasing granularity. This allows us to include minute densities in 
+the tail of the distribution without having to exceed four digits of 
+precision. For example, LCP values less than 3 seconds are grouped into bins 
+200ms wide. Between 3 and 10 seconds, bins are 500ms wide, then 5000ms wide, 
+etc. Rather than having bins of varying widths, bin spreading ensures that all 
+bins are a constant 100ms wide, and the distribution is linearly interpolated 
+across each bin.
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:173]: We just said that bins go from 200ms wide then 500ms wide then 5000ms wide but here we're saying they are a constant 100ms wide

Since these are technical details you can leave this as is if you think it's a good description of what's going on, I'm just letting you I didn't get it on my first readthrough

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md [hunk 12]
```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
...
+
+<img class="w-screenshot" src="./dash_device.png" alt="CrUX Dashboard device page">
+
+Similarly, the ECT distribution page shows you the breakdown of 4G, 3G, 2G, 
+slow 2G, and offline experiences.
+
+{% Aside 'key-term' %}
+Effective connection types are considered _effective_ because they're based 
+on bandwidth measurements on users' devices, and don't imply any particular 
+technology used. For example, a desktop user on fast WiFi may be labelled as
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:198]:
```suggestion
technology used. For example, a desktop user on fast Wi-Fi may be labelled as 
```

File: src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md [hunk 13]
```diff
@@ -7,15 +7,15 @@ description: |
...
 
 [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/)
 (PSI) is a tool for web developers to understand what a page's performance is
 and how to improve it. It uses [Lighthouse](https://developers.google.com/web/tools/lighthouse/)
-to audit a given page and identify opportunities to improve performance. It also
-integrates with the Chrome UX Report ([CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/))
-to show how real users experience performance on the page and the origin in
+to audit the page and identify opportunities to improve performance. It also
+integrates with the [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report/) (CrUX)
+to show how real users experience the page and the origin in
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md:18]:
```suggestion
to show how real users experience the page and the [origin](/same-site-same-origin/#origin) in
```

File: src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md [hunk 14]
```diff
@@ -35,85 +35,96 @@ controlled test in the lab.
...
 
-In the Field Data section, you'll see two metrics: First Contentful Paint (FCP)
-and First Input Delay (FID). These metrics are representative of the user
-experience in different ways. FCP measures the time until the page displays
-something in the foreground, like some text or a logo. FID measures the
-interactivity of the page, from the user's first interaction to the time the
-page responds to it.
-
-Metric | Fast | Average | Slow
+In the Field Data section, you'll see four metrics: [First Contentful Paint](/fcp/)
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md:38]:
```suggestion
In the **Field Data** section, you'll see four metrics: [First Contentful Paint](/fcp/)
```

File: src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md [hunk 15]
```diff
@@ -35,85 +35,96 @@ controlled test in the lab.
...
+assessment. To pass, the percentile must be categorized as "good".
+Otherwise, the page is labelled as "not passing".
+
+The percentiles shown for all metrics correspond to the 75th percentile. In
+statistics, a percentile is a measure that indicates the value below which a
+given percentage of samples fall. For example, the screenshot above shows that
+FID's 75th percentile is 15ms, meaning that 75% of FID experiences are faster
+than 15ms. These values are color-coded according to the threshold table above
+where good values are green, values needing improvement are orange, and poor
 values are red.
```
Comment (kaycebasques) [inline src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md:81]: It might be good to consistently wrap the 3 evaluation keywords consistently in quotes. E.g. "good" and "needs improvement", and "poor". And to do this consistently throughout all updated docs.

File: src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md (hunk 11 above)
Comment (rviscomi) [reply src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md:173]: Added a parenthetical about GCD

File: src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md (hunk 13 above)
Comment (rviscomi) [reply src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md:20]: Hmm none of the origins in CrUX specify a port and this link might lead some readers to think that they need to include one when looking up an origin.
